packages/react/toast/src/Toast.stories.tsx

```diff
@@ -549,7 +559,7 @@ const buttonClass = css({
   border: '1px solid black',
   borderRadius: 4,
   background: 'gainsboro',
-  font: 'inherit',
```

Comment:
This is the cause of the chromatic diff prompt, for some reason `font` wasn't rendering in the previous Stitches version (we usually use `fontFamily`)

I've updated it to use `fontFamily` just for consistency with the other stories though it's functionally the same now anyway since it was seemingly fixed.